addons/mrp_landed_costs/models/stock_landed_cost.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+
+
+class StockLandedCost(models.Model):
+    _inherit = 'stock.landed.cost'
+
+    target_model = fields.Selection(selection_add=[('manufacturing', "Manufacturing Orders")])
+    mrp_production_ids = fields.Many2many(
+        'mrp.production', string='Manufacturing order',
+        copy=False, states={'done': [('readonly', True)]}, groups='mrp.group_mrp_user')
```

Comment:
You need to be stock manager in order to have right on landed cost. You should probably also be a mrp manager in order to register landed cost

Reply:
cc @oma-odoo un avis?